shadowsocks-csharp/Controller/System/AutoStartup.cs

```diff
@@ -14,7 +14,7 @@ static class AutoStartup
 
         // Don't use Application.ExecutablePath
         // see https://stackoverflow.com/questions/12945805/odd-c-sharp-path-issue
-        
+
         private static string Key = "Shadowsocks_" + Program.ExecutablePath.GetHashCode();

```

Comment:
string.GetHashCode() are not guranteed same between different versions, it's the possible reason why there's two different entry.

Reply:
true, the other issue is that the source string (which we calculate hash of) has been changed from https://github.com/shadowsocks/shadowsocks-windows/commit/ced49e2de12b70908c3ce43a33cb4130fa177879